reactioncommerce/reaction-next-starterkit
# (feat): New Profile Address page

Resolves #245 
Impact: **minor**
Type: **feature**

## Issue
Create a profile address book page

## Solution
Create a profile address book page. This includes adding a new route, `/profile/address`.

## Breaking changes
None

## Testing

- Visit `/profile/address` as a non-logged in user, and see a 404 / page not found
- Visit `/profile/address` as a logged in use, and see a page that matches our design: https://zpl.io/2vYPqk7
- - This page includes 3 new components (`AddressBook`, `InPageMenuItem`, `ProfileInfo`), and various other new components inside these new components.
- Confirm new "Profile" button in user menu when logged in
- Confirm you can view, add, update, and delete addresses.
